docs/search/index.asciidoc

```diff
@@ -19,53 +19,53 @@ The *Search* space in {kib} comprises the following features:
 
 The Search solution and use case is made up of many tools and features across the {stack}.
 As a result, the release notes for your features of interest might live in different Elastic docs.
-// Use the following table to find links to the appropriate documentation, API references (if applicable), and release notes.
+Use the following table to find links to the appropriate documentation, API references (if applicable), and release notes.
 
-// [options="header"]
-// |===
-// | Name               | API reference           | Documentation         | Release notes
+[options="header"]
+|===
+| Name               | API reference           | Documentation         | Release notes
 
-// | Connectors
-// | link:https://example.com/connectors/api[API reference]
-// | link:https://example.com/connectors/docs[Documentation]
-// | link:https://example.com/connectors/notes[Release notes]
+| Connectors
+| link:https://www.elastic.co/guide/en/elasticsearch/reference/8.15/connector-apis.html[API reference]
+| <<action-types,Documentation>>
+| link:https://www.elastic.co/guide/en/elasticsearch/reference/current/es-release-notes.html[Elasticsearch guide]
```

Comment:
```suggestion
| {ref}/es-connectors[Elastic Connectors]
| {ref}/es-connectors-release-notes.html[Elasticsearch guide]
```

Reply:
action-types are Kibana connectors (for LLM providers usually) 🤯 

I moved connectors docs for 8.16.0 so there was no way for you to know the release notes had a new URL :)

Reply:
Should use `{ref}` syntax instead of full URLs `https://www.elastic.co/guide/en/elasticsearch/reference/current/`

That way links will resolve to the correct version branches. 

If you ever need to check these variables (known as attributes in asciidoc) we have a shared file for them here: https://github.com/elastic/docs/blob/master/shared/attributes.asciidoc

This comment applies to all links in this PR.

Reply:
@kosabogi this comment still applies:

> Should use {ref} syntax instead of full URLs https://www.elastic.co/guide/en/elasticsearch/reference/current/

Need to replace all hardcoded URLs as much as possible with those attribute values